stress/shell/src/main/java/alluxio/stress/cli/MasterBatchTask.java

```diff
@@ -0,0 +1,130 @@
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.stress.BaseParameters;
+import alluxio.stress.master.MasterBatchTaskParameters;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParametersDelegate;
+import com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Multiple tasks runner for Master StressBench.
+ */
+public class MasterBatchTask extends BatchTask {
+  private static final Logger LOG = LoggerFactory.getLogger(MasterBatchTask.class);
+  // Target throughput is set to a huge number to reach the maximum throughput of the task.
+  private static final int TARGET_THROUGHPUT  = 1_000_000;
+
+  @ParametersDelegate
+  private BaseParameters mBaseParameter = new BaseParameters();
+  @ParametersDelegate
+  private MasterBatchTaskParameters mParameter = new MasterBatchTaskParameters();
+
+  @Override
+  public void run(String[] args) {
+    JCommander jc = new JCommander(this);
+    jc.setProgramName(this.getClass().getSimpleName());
+    try {
+      jc.parse(args);
+    } catch (Exception e) {
+      LOG.error("Failed to parse command: ", e);
+      System.out.println(getDescription());
+      throw e;
+    }
+
+    if (mParameter.mWriteType.equals("ALL")) {
+      System.err.format("Parameter write-type ALL is not supported in batch task %s",
+          mParameter.mTaskName);
+      return;
+    }
+
+    if (mParameter.mStopCount != -1 && !(mParameter.mWarmup.equals("0")
+        || mParameter.mWarmup.equals("0s"))) {
+      System.out.println("Warning: when using stop-count parameter to claim the files "
+          + "to be operated, the operations executed in warmup time will not be calculated in"
+          + "the final result. The calculated operations will be less than expected.");
+    }
+
+    List<String[]> command = getCommand();
+    for (String[] arg : command) {
+      System.out.println("-----------------------------------------------------");
+      System.out.format("Now executing command : %s on MasterStressBench...%n", arg[1]);
+      // If one individual task fails, the pipeline will try to move on and finish
+      try {
+        StressMasterBench bench = new StressMasterBench();
+        String jsonResult = bench.run(arg);
+        System.out.println("Task finished successfully. The result is following :");
+        System.out.println(jsonResult);
+      } catch (Exception e) {
+        System.err.format("Failed to finish the %s operation%n", arg[1]);
+        e.printStackTrace();
+      }
+    }
+    System.out.println("-----------------------------------------------------");
+    System.out.println("All tasks finished. You can find the test results in the outputs above.");
+  }
+
+  private List<String[]> getCommand() {
+    List<String[]> commandList = new ArrayList<>();
+
+    if (mParameter.mTaskName.equals("MasterComprehensiveFileBatchTask")) {
```

Comment:
"MasterComprehensiveFileBatchTask" can be a `private static final string COMPREHENSIVE_FILE_BATCH_TASK_NAME`